pkg/controllers/management/rkeworkerupgrader/upgrade.go

```diff
@@ -230,26 +230,34 @@ func (uh *upgradeHandler) upgradeCluster(cluster *v3.Cluster, nodeName string, p
 	defer uh.clusterLock.Unlock(clusterName)
 
 	logrus.Debugf("cluster [%s] upgrading condition: [%v] plan changed: [%v]", clusterName, v3.ClusterConditionUpgraded.IsUnknown(cluster), planChanged)
+
+	var (
+		clusterCopy *v3.Cluster
+		err         error
+	)
 	if !v3.ClusterConditionUpgraded.IsUnknown(cluster) || planChanged {
 		clusterCopy := cluster.DeepCopy()
```

Comment:
since we continue to use same clusterCopy object line 244 onwards, `clusterCopy := cluster.DeepCopy()` should be changed to `clusterCopy = cluster.DeepCopy()`?